cmake/package.cmake

```diff
@@ -4,26 +4,65 @@ set_target_properties(
 )
 install(TARGETS
   onnxruntime-genai
-  LIBRARY
-  PUBLIC_HEADER
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION lib
```

Comment:
without doing this onnxruntime-genai.dll will be in bin folder

Reply:
But, why it should not be in the bin folder?

Reply:
I thought ORT has the dll in lib, but if you think we can put it in bin folder, that is fine too.